docs/content-contribution/overall-process.md

```diff
@@ -8,7 +8,7 @@ The overall process for content contribution starts with you.
 
 1. You're reading documentation on [SAP Help Portal](http://help.sap.com). You come across something you're not sure is entirely accurate, or you've spotted a typo, or you think that there's a small addition or amendment that you can offer that would help clarify or otherwise add value to what's already there.
 
-2. You look for the **Edit** button in the function bar at the top of the page. If there is no **Edit** button, the documentation is not yet part of SAP's Open Documentation Initative.
+2. You look for the *Edit page* option via the *Feedback* link in the function bar near the top of the page. If there is no such option, the documentation is not yet part of SAP's Open Documentation Initative.
```

Comment:
If we use "link", readers might look for blue underlined text. I think "menu" makes more sense.  You also use menu [here](https://github.com/SAP-docs/contribution-guidelines/blob/ca12562f901417d5b37388d6eeb60ecf73e88371/docs/contributing.md) already.

```suggestion
2. You look for the *Edit page* option via the *Feedback* menu in the function bar near the top of the page. If there is no such option, the documentation is not yet part of SAP's Open Documentation Initiative.
```